ecs.tf

```diff
@@ -93,6 +101,25 @@ resource "aws_s3_bucket_object" "prometheus_config" {
   kms_key_id = data.terraform_remote_state.management.outputs.config_bucket.cmk_arn
 }
 
+resource "aws_service_discovery_private_dns_namespace" "prometheus" {
```

Comment:
As this is a new resource type (I think) then let's just be prepared to have to open up the web proxy to allow access to this and the service below's API endpoints from CI. They don't have VPC endpoints at the moment.

Reply:
Added to whitelist on internet-egress repo